docs/site/Sequence.md

```diff
@@ -71,6 +129,13 @@ class MySequence extends DefaultSequence {
 }
```

Comment:
Two lines above, you are calling `await this.send(...)`. Please make sure we are consistently showing the same approach in all places in our docs.